.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
+- **Brief description of the command being documented:** unspecified
+<!-- Replace `latest` with a concreete version when you know it exactly. -->
+
+- [x] New command pages or translations are added. <!-- Uncheck if you modify Tl;Dr tooling such as CI. -->
+- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`. <!-- Check when you are sure in it. -->
```

Comment:
This comment really add information for the collaborator?
```suggestion
- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

Reply:
I am not sure. 🤔